bindings/rust/src/lib.rs

```diff
@@ -40,14 +40,65 @@ mod sys;
 use std::ffi::CString;
 use std::ptr::NonNull;
 
+/// A safe container for handling the low-level FizzyError struct.
+struct CErrorContainer(Box<sys::FizzyError>);
```

Comment:
Any better name for this? CE looks like the OE letter.

Reply:
`FizzyErrorContrainer`, `FizzyErrorBox`

Reply:
Changed.